test/tests/can.c

```diff
@@ -0,0 +1,133 @@
+/* CAN test
+
+- Setups up CAN1 in loopback mode
+- Send 4 messages (since there are 3 mailboxes, 1 ends up going in the can1 send queue)
+- Recieves the 4 messages and verifies correctness
+- Flashes LED if successful
+*/
```

Comment:
I really like this test. I think all of our tests should look like this. It clearly explains what it does and it's thoroughly commented